File: Practica-POO/Semana01/Ejercicio08/Clock.py
class Clock:
    def __init__(self,hours,minutes,seconds):
        self.hours = hours
        self.minutes = minutes
        self.seconds = seconds

    def horario(self):
        return f"{self.hours:02d}:{self.minutes:02d}:{self.seconds:02d}"

    def TotalSeconds(self):
        if (self.hours<=12):
           return (self.hours*3600)+(self.minutes*60)+self.seconds
        else:
           return ((self.hours-12)*3600)+(self.minutes*60)+self.seconds

clock1 = Clock(13,50,3)

hora1         = clock1.horario()
totalseconds1 = clock1.TotalSeconds()

print("---Hora---\n","\t",hora1 )
print("---Total Seconds---\n","\t",totalseconds1)



    

File: Practica-POO/Semana01/Ejercicio08/Clock.cpp
#include <iostream>
#include <string>
using namespace std;

class Clock{
    public:
    int hours;
    int minutes;
    int seconds;

    Clock(int h , int m , int s){
        hours = h;
        minutes = m;
        seconds = s;
    }

    string Horario(){
        return to_string(hours) + ":" + to_string(minutes) + ":" + to_string(seconds);
    }

    int TotalSeconds(){
        return (hours*3600)+(minutes*60)+seconds;
    }
};

int main(){
    Clock clock1(9,59,46);

    string hora1 = clock1.Horario();
    int totalseconds1 = clock1.TotalSeconds();
    
    cout << "Hora\n" <<"\t" << hora1 <<endl;
    cout << "Segundos totales\n" << "\t" << totalseconds1 << endl; 
    return 0;
}